cirq-google/cirq_google/__init__.py

```diff
@@ -127,6 +127,10 @@
     SerializableGateSet,
 )
 
+from cirq_google.experimental.ops import (
```

Comment:
This essentially makes this feature "public" - how about just keeping it public within `cirq_google.experimental`? `experimental` is already imported to the `cirq_google` namespace.

Reply:
Also,`cirq_google.experimental` is setup in the json testing framework as well.

Reply:
That's fine, but it does imply an eventual breakage if we ever move it out of experimental for anyone using it.  Given that it is in experimental, I suppose that's okay, but it will inconvenience a handful of researchers.

Reply:
Done.

Reply:
Yeah, we can then deprecate the experimental reference and gracefully transition. It might just be dropped eventually as well, and then that will be the deprecation. In the meantime though we don't inconvenience other users of cirq_google by adding it to the main namespace.